libsolidity/interface/StandardCompiler.cpp

```diff
@@ -111,7 +111,8 @@ Json::Value formatErrorWithException(
 	bool const& _warning,
 	string const& _type,
 	string const& _component,
-	string const& _message
+	string const& _message,
+	ErrorId _errorId = ErrorId()
```

Comment:
Shouldn't this be `std::optional<ErrorId>`?

Reply:
Why not :), b0529c6.